src/modules/fw_att_control/ecl_controller.cpp

```diff
@@ -0,0 +1,145 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2013 Estimation and Control Library (ECL). All rights reserved.
```

Comment:
@dagar is it okay to keep the wrong header for now? Or should we at least change them, plus maybe also the file names (remove the "ecl" from them?)

Reply:
I think it's fine. They'll also be refactored out of existence at some point.